src/js/containers/covid19/awardSpendingAgency/AwardSpendingAgencyTableContainer.jsx

```diff
@@ -293,6 +293,12 @@ const AwardSpendingAgencyTableContainer = (props) => {
         window.scrollTo(0, 0);
     }, [document]);
 
+    useEffect(() => {
```

Comment:
Take a look at the documentation [here](https://reactjs.org/docs/hooks-effect.html#effects-with-cleanup). What we want is for this useEffect to be fired when the component unMounts. Right now, this is gonna fire onMount.